src/ui/index.css

```diff
@@ -1,493 +1,136 @@
 @import "../../node_modules/normalize.css/normalize.css";
+@import "../../node_modules/bootstrap/dist/css/bootstrap.min.css";
+
+:root {
+    --colour-primary: #6a3d72;
+    --colour-secondary: #44144c;
+    --colour-tertiary: #281548;
+    --colour-primary-light: #f0eaf0;
+    --colour-primary-medium: #c7bed4;
+    --colour-gray: #D4D0D0;
+    --box-shadow-hover: rgba(0, 0, 0, 0.2);
+}
 
 body, html {
-    margin: 0;
-    padding: 0;
     font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
 }
 
 .navbar {
-    background-color: #44144c;
-    padding: 16px 0;
-}
-
-.container {
-    width: 100%;
-    margin: auto;
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-}
-
-.nav__content {
-    display: flex;
-}
-
-.nav__logo {
-    height: 30px;
-    margin-right: 10px;
-    padding-left: 20px;
+    background-color: var(--colour-secondary);
 }
 
-.nav__link {
-    font-size: 1.5rem;
-    font-weight: 600;
-    color: #f3f2f3;
-    text-decoration: none;
-    padding-left: 5px;
+.small-font {
+    font-size: 0.8rem !important;
```

Comment:
We shouldn't need any `!important` overrides for rules, since all of our custom rules are last in the cascade.